### DISCUSSION TITLE
如何修改管理器在安装模块后跳出的通知？

### DISCUSSION THREAD

**Discussion by 20171213** [LSPosed/LSPosed#2450] [Q&A]
类原生安卓11，改了通知为静音且最小化后再安装模块还是会弹出悬浮通知并播放通知

**Comment by Kprobem** [comment]
Toast还是Dialog?

**Reply by 20171213** [reply]
> Toast还是Dialog?

安装后会在通知面板显示一个无法关闭的"Lsposed已加载“通知，长按进入系统通知管理也无法关闭

**Comment by yeezylife** [comment]
同问，我长按=>关闭通知=>LSPosed状态，点掉，完成。 然后就消失了，但是下次打开屏幕通知栏的这个状态条"Lsposed已加载”又出现了。。。

**Comment by tianjyan** [comment]
创建这个通知的程序是app_process运行的，属于system_server的进程，正常的系统是无法屏蔽系统的通知，要想不显示这个通知可以在程序里设置创建快捷键或者构建自己的LSPosed版本。

参考：
https://github.com/LSPosed/LSPosed/blob/b07fd258d20c4a864975776f27167eb1a3726494/daemon/src/main/java/org/lsposed/lspd/service/LSPNotificationManager.java#L152-L176
https://github.com/LSPosed/LSPosed/blob/b07fd258d20c4a864975776f27167eb1a3726494/daemon/src/main/java/org/lsposed/lspd/service/LSPManagerService.java#L613-L620
